Expecta/Matchers/EXPMatchers+postNotification.m

```diff
@@ -55,29 +53,29 @@ - (BOOL)exp_isFunctionallyEqualTo:(NSNotification *)otherNotification
     return YES;
   });
 
-  match(^BOOL{
+  match(^BOOL(id actual){
```

Comment:
`actual` is not used in the block. Should it be marked with `__unused`? 
In addition, add space.

Reply:
This project doesn't trigger warnings on such things (yet!). I'll add `__unused` though.